lib/actions/api.js

```diff
@@ -45,21 +47,88 @@ export function planTrip (customOtpQueryBuilder) {
   }
 }
 
-// function setURLSearch (params) {
-//   window.location.hash = `#plan?${params.split('plan?')[1]}`
-// }
+// Profile Query
+
+export const receivedProfileError = createAction('PROFILE_ERROR')
+export const receivedProfileResponse = createAction('PROFILE_RESPONSE')
+export const requestProfileResponse = createAction('PROFILE_REQUEST')
```

Comment:
I sort of wonder if it may be a bit redundant to have these actions for profile requests/responses.  Perhaps we can just re-use the existing request/response actions but still have different logic for making a profile vs itinerary request.